WorkloadManagementSystem/Agent/SiteDirector.py

```diff
@@ -1349,7 +1349,7 @@ def _updatePilotStatus(self, pilotRefs, pilotDict, pilotCEDict):
 
       if newStatus:
         self.log.info('Updating status to %s for pilot %s' % (newStatus, pRef))
-        result = pilotAgentsDB.setPilotStatus(pRef, newStatus, statusReason='Updated by SiteDirector')
+        result = pilotAgentsDB.setPilotStatus(pRef, newStatus, '', 'Updated by SiteDirector')
```

Comment:
is the setPilotStatus method has changed? Why you need this modification?

Reply:
This is a usual case where either service or direct db access is used depending on the db accessibility. The changed call was working with the database but not with the service which is not allowing keyword arguments. This fix makes both kinds of the call working